SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -5315,6 +5478,27 @@ void CEulerSolver::Upwind_Residual(CGeometry *geometry, CSolver **solver_contain
       
       Dissipation_i = node[iPoint]->GetRoe_Dissipation();
       Dissipation_j = node[jPoint]->GetRoe_Dissipation();
+
+      if (config->GetKind_HybridRANSLES() == MODEL_SPLIT) {
+	// increase dissipation if resolution adequacy says we can't resolve
+	const su2double rMi = solver_container[FLOW_SOL]->average_node[iPoint]->GetResolutionAdequacy();
+	const su2double rMj = solver_container[FLOW_SOL]->average_node[jPoint]->GetResolutionAdequacy();
+	if ( rMi > 1.0) {
+	  //Dissipation_i = max(Dissipation_i, 0.1);
+	  //Dissipation_i += 2.0*tanh(log10(rMi));
+	  Dissipation_i += tanh(log10(rMi));
+	  Dissipation_i = min(Dissipation_i, 1.0);
+	  node[iPoint]->SetRoe_Dissipation(Dissipation_i);
+	}
+	if ( rMj > 1.0 ){
+	  //Dissipation_j += 2.0*tanh(log10(rMj));
+	  Dissipation_j += tanh(log10(rMj));
+	  Dissipation_j = min(Dissipation_j, 1.0);
+	  node[jPoint]->SetRoe_Dissipation(Dissipation_j);
+	}
+
+      }
+
```

Comment:
git bisect plus some additional testing indicates that this nonsense is creating problems for the channel test case.  The goal of these changes was to increase the upwinding in regions where the resolution adequacy parameter is not "small".  This was in response to spurious oscillations that were observed at the boundary layer edge in the axisymmetric bump case that would sometimes become non-physical (negative density, temperature, etc).  However, it was only marginally successful in that case.  Given that it is causing problems for the channel case, these changes should not be included in the PR.  I will prepare a commit removing them.  We can investigate the effect on the axisymmetric bump separately.